src/archetype.rs

```diff
@@ -97,17 +100,24 @@ impl Archetypes {
     pub(crate) fn spawn(&mut self, id: EntityId) -> EntityLocation {
         let empty = self.empty_mut();
 
-        let rellocated = unsafe { empty.reserve_one() };
+        // Reserve space for the spawned entity.
```

Comment:
This should probably have a [`// SAFETY` comment](https://std-dev-guide.rust-lang.org/policy/safety-comments.html)